src/test/resources/metaModel/address_book_city.json

```diff
@@ -6,9 +6,20 @@
         "info": "Schema for storing city information",
         "entities": [
           {
+            "proto3_": {
+              "path": "something",
+              "options": [
+                "(custom_options.ctx) = cluster"
+              ]
+            },
             "name": "address_book_city",
             "fields": [
               {
+                "proto3_": {
+                  "options": [
+                    "Test single inline option"
```

Comment:
Use valid options so that `protoc` won't complain when we test the generated file with it.